packages/erebos-api-bzz-node/src/index.js

```diff
@@ -127,6 +128,20 @@ export default class Bzz extends BaseBzz {
     )
   }
 
+  uploadDirectoryTar(path: string): Promise<string> {
+    const readStream = tarFs.pack(path)
+
+    return this._fetch(`${this._url}bzz:`, {
+      method: 'POST',
+      body: readStream,
```

Comment:
Not an issue but the code could be even simpler by setting `body: tarFs.pack(path)` directly as it's the only use of the extra variable.